ui/components/issuesidebar/issuesidebar.go

```diff
@@ -97,6 +151,14 @@ func (m Model) View() string {
 		s.WriteString(m.commentBox.View())
 	}
 
+	if m.isAssigning {
+		s.WriteString(m.assignBox.View(true))
```

Comment:
In bubbletea, state like this (whether we're assigning or unassining) is held on the model - in this case the `assignbox`'s model.
To update this value, you usually just do `m.assignBox.setAssignState(AssignBox.Assigning)` or something like that.
The `View` method then reads the model and renders accordingly